micromanager_gui/_signals.py

```diff
@@ -0,0 +1,9 @@
+from qtpy.QtCore import QObject, Signal
+
+__all__ = [
+    "main_window_events",
+]
+
+
+class main_window_events(QObject):
+    availableChannelsChanged = Signal(tuple)
```

Comment:
Introduction of an events object for the widget is probably the biggest change here